apps/www/_blog/2022-10-03-postgres-wasm.mdx

```diff
@@ -0,0 +1,242 @@
+---
+title: Postgres WASM by Snaplet and Supabase
+description: Turns out the question of which identifier to use as a Primary Key is complicated -- we're going to dive into some of the complexity and inherent trade-offs, and figure things out
```

Comment:
Sanity check: is this the intended description? Seems like the description for the prev blog post

Reply:
It's not yeah. Will update it.